community/bolt/src/docs/dev/serialization.asciidoc

```diff
@@ -422,7 +422,7 @@ Path (signature=0x50) {
 }
 ----
 
-The two lists `N` and `R` are defined as follows:
+The two lists `N` and `R` (short for `nodes` and `relationships` in the example above) are defined as follows:
```

Comment:
The section on `Paths` is pretty tricky, took me a few reads to understand. I am not sure how to make it more accessible but maybe it should be revisited at some point